project/Dependencies.scala

```diff
@@ -124,7 +127,8 @@ object Dependencies {
 
   lazy val httpTestkit = l ++= Seq(
     Test.junit, Test.junitIntf, Compile.junit % "provided",
-    Test.scalatest.value.copy(configurations = Some("provided; test"))
+    Test.scalatest.value.copy(configurations = Some("provided; test")),
+    Test.specs2.value
```

Comment:
Can we keep the style consistent with scalatest here? I.e. keep the config as `"test"` in the `Test` object above and use `copy` here.